specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/FormRecognizer.json

```diff
@@ -0,0 +1,2344 @@
+        },
+        "/prebuilt/receipt/analyzeResults/{resultId}": {
+            "get": {
+                "summary": "Get Analyze Receipt Result",
+                "description": "Track the progress and obtain the result of the analyze receipt operation.",
+                "operationId": "GetAnalyzeReceiptResult",
+                "consumes": [],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "resultId",
+                        "in": "path",
+                        "description": "Analyze operation result identifier.",
+                        "required": true,
+                        "type": "string",
+                        "format": "uuid"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Success",
+                        "schema": {
+                            "$ref": "#/definitions/AnalyzeOperationResult"
+                        }
+                    },
+                    "default": {
+                        "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Get analyze receipt result": {
+                        "$ref": "./examples/ReceiptsBatchResult.json"
+                    }
+                }
+            }
+        },
+        "/layout/analyze": {
+            "post": {
+                "summary": "Analyze Layout",
+                "description": "Extract text and layout information from a given document. The input document must be of one of the supported content types - 'application/pdf', 'image/jpeg', 'image/png', 'image/tiff' or 'image/bmp'. Alternatively, use 'application/json' type to specify the location (Uri or local path) of the document to be analyzed.",
+                "operationId": "AnalyzeLayoutAsync",
+                "consumes": [
+                    "application/pdf",
+                    "application/json",
+                    "image/jpeg",
+                    "image/png",
+                    "image/tiff",
+                    "image/bmp"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/FileStream"
+                    },
+                    {
+                        "$ref": "#/parameters/Pages"
+                    },
+                    {
+                        "$ref": "#/parameters/Language"
+                    },
+                    {
+                        "$ref": "#/parameters/ReadingOrder"
```

Comment:
Is reading order only going to be on analyze layout?

Reply:
That is the plan for v2.1-preview.3.  We would like to gather additional customer feedback before expanding its availability.

Reply:
Ok good to know, thanks for the info!